docs/_docs/framework/capabilities/optimize/sustainability.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+grand_parent: FinOps Framework
+parent: Optimize
+title: Cloud sustainability
+permalink: /framework/capabilities/optimize/sustainability
+nav_order: 1
+description: This article helps you understand the cloud sustainability capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+author: bandersmsft
+ms.author: banders
+ms.date: 04/09/2024
+ms.topic: conceptual
+ms.service: finops
+ms.reviewer: micflan
+---
+
+<span class="fs-9 d-block mb-4">Cloud sustainability</span>
+This article helps you understand the cloud sustainability capability within the FinOps Framework and how to implement that in the Microsoft Cloud.
+{: .fs-6 .fw-300 }
+
+<details open markdown="1">
+  <summary class="fs-2 text-uppercase">On this page</summary>
+
+- [▶️ Getting started](#️-getting-started)
+- [🏗️ Building on the basics](#️-building-on-the-basics)
+- [🍎 Learn more at the FinOps Foundation](#-learn-more-at-the-finops-foundation)
+- [⏩ Next steps](#-next-steps)
+- [🧰 Related tools](#-related-tools)
+
+</details>
+
+---
+
+<a name="definition"></a>
+**Cloud sustainability defines how organization's decision-making process regarding cloud usage taking into account its environmental impact and aligning with broader sustainability objectives."**
+{: .fs-6 .fw-300 }
+
+It delineates policies, guidelines, and strategies to optimize cloud sustainability, identifying improvement opportunities and fostering stakeholder engagement for effective evaluation. 
+
+<br>
+
+## ▶️ Getting started
+
+When you first start with cloud sustainability, there are a few built-in tools that can help you 
+
+- To learn more about what environmental, social, and governance (ESG) capabilities Microsoft offers across the cloud portfolio and solutions, refer to the [Microsoft Cloud for Sustainability.](https://learn.microsoft.com/industry/sustainability/overview)
+-Review the [Microsoft Cloud for Sustainability design principles](https://learn.microsoft.com/industry/well-architected/sustainability/cloud-sustainability-design-principles) outlined as part of the Well-Architected for Sustainability guidance.
+- Use the [Azure carbon optimization](https://learn.microsoft.com/azure/carbon-optimization/overview) to find opportunities to optimize your Azure environment to lower your carbon emissions and costs.
+- Calculate your cloud-based carbon emissions with the [Emissions Impact Dashboard for Azure](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard) or the [Emissions Impact Dashboard for Microsoft 365](https://learn.microsoft.com/power-bi/connect-data/service-connect-to-emissions-impact-dashboard).
+- Review the Microsoft Cloud for Sustainability [reference architectures](https://learn.microsoft.com/industry/well-architected/sustainability/sustainability-architecture-overview) to identify which tools are more relevant for your environmental, social, and governance (ESG) capabilities.
+<br>

```

Comment:
Should we also highlight the Sustainability section of the Advisor Cost Optimization workbook?